wrc/codegen/cghtml.py

```diff
@@ -94,7 +94,7 @@ def __init__(self, versionhash, language, pdf, merged=False):
         super(WCADocumentHtml, self).__init__(str)
         self.regset = set()
         if merged:
-            self.urls = {'regulations': './full', 'guidelines': './full', 'pdf': pdf}
+            self.urls = {'regulations': './', 'guidelines': './', 'pdf': f'../{pdf}'}
```

Comment:
why does the PDF need the `../` backwards navigation?
That's not happening in the "non-merged" documents, why does the merged version suddenly need it?

Reply:
@gregorbg having both PDFs sounds good to you?